src/main/kotlin/no/nav/yrkesskade/ysmeldingapi/services/SkademeldingService.kt

```diff
@@ -61,4 +67,44 @@ class SkademeldingService(private val skademeldingInnsendingClient: Skademelding
         // returner lagrede skademelding
         return lagretSkademeldingDto
     }
+
+    private fun validerSkademelding(skademelding: Skademelding) {
+        // Hent kodelister basert på rolletype som skal benyttes for å finne gyldige verdier
+        check(skademelding.skadelidt!!.dekningsforhold.rolletype == null, { "rolletype er påkrevd" })
```

Comment:
Jeg tror kanskje denne bør sjekke for `!= null`? Evt. bruke `checkNotNull`?

Reply:
Fikset! Jeg la også på litt flere checks